### PR TITLE
chore: starting to test filter types

### PR DESCRIPTION
**Pull request by tylersayshi**

### CODE REVIEW

File: examples/react/filters/src/main.tsx [hunk 1]
```diff
@@ -42,6 +43,8 @@ const _features = tableFeatures({
   rowPaginationFeature,
 })
 
+type TestType = ExtractFeatureTypes<typeof _features, 'Column'>
```
Comment (tylersayshi) [inline examples/react/filters/src/main.tsx:46]: This is currently unknown. It seems like more needs to be done to override the interface of the Columns maybe, is this correct?

**Comment by KevinVandy** [comment]
Where would attest be installed?

**Comment by KevinVandy** [comment]
maybe another branch that actually uses the `ExtractFeatureTypes` should be made? We'll uncomment that code in that branch, run attest on both the alpha branch and that one.

**Comment by tylersayshi** [comment]
> Where would attest be installed?

Probably the root package.json devDeps. This way we can test examples, packages/* and anything else from there.

I don't have any strong opinion on this though. I did a global install of it locally and I am just sampling as I `cd` around the repo right now.

**Comment by tylersayshi** [comment]
> maybe another branch that actually uses the `ExtractFeatureTypes` should be made? We'll uncomment that code in that branch, run attest on both the alpha branch and that one.

yeah, this would be really helpful. Is it just a matter of uncommenting all the commented out types? 

I was unsure if not all of them were commented for the same reasons or not.

**Comment by KevinVandy** [comment]
assuming we want to close this one

**Pull request closed without merging**